server/src/daemon/error.rs

```diff
@@ -0,0 +1,21 @@
+// Errors
+
+#[derive(zbus::DBusError, Debug)]
+#[zbus(prefix = "org.freedesktop.Secret.Error")]
+pub enum ServiceError {
```

Comment:
The exact same error is defined https://github.com/bilelmoussaoui/oo7/blob/main/client/src/dbus/error.rs#L6 already, make use of that instead

Reply:
done, thanks for the prompt review.